test/ShallowWrapper-spec.jsx

```diff
@@ -86,6 +90,22 @@ describe('shallow', () => {
         expect(wrapper.context().name).to.equal(context.name);
         expect(wrapper.context('name')).to.equal(context.name);
       });
+
+      itIf(REACT16, 'is not introspectable through context API', () => {
```

Comment:
why is this the case? if you can inspect the props passed to an SFC, why not the context?